opencti-platform/opencti-graphql/src/domain/stixCyberObservable.js

```diff
@@ -194,6 +194,11 @@ export const addStixCyberObservable = async (context, user, input) => {
   if (!input[graphQLType]) {
     throw FunctionalError(`Expecting variable ${graphQLType} in the input, got nothing.`);
   }
+  const lowerCaseTypes = ['Domain-Name', 'Url', 'Email-Addr'];
```

Comment:
Do we really want URL here ? See https://github.com/OpenCTI-Platform/opencti/issues/7311#issuecomment-2223003003

Reply:
Kinda validated it as "good candidat" with Sam yesterday, but I will remove it for now until we're sure. Especially after this comment. thanks for sharing it